services/permits/app/compare_extraction_results.py

```diff
@@ -1,18 +1,21 @@
 ###
 # Utility script to compare extracted permit conditions from CSV files to generate a csv and html report of how well they match
 # Usage: python compare_extraction_results.py --csv_pairs <auto_extracted_csv> <manual_extracted_csv> --csv_pairs <auto_extracted_csv> <manual_extracted_csv> ...
```

Comment:
Just included a couple of tweaks to the report generation to include the extracted `meta` dict in the report html